File: app/src/main/java/huce/fit/appreadstories/story/filter/StoryFilterPresenter.java
package huce.fit.appreadstories.story.filter;

import huce.fit.appreadstories.story.BaseListStoryPresenter;

public interface StoryFilterPresenter extends BaseListStoryPresenter {

    void setSpeciesId(int mSpeciesId);
    void setStatusId(int mStatusId);
    int getSpeciesId();
    int getStatusId();
    void setSpecies(String species);
    void setStatus(String status);
    void getData();
}

File: app/src/main/java/huce/fit/appreadstories/story/list/filter/StoryFilterPresenter.kt
package huce.fit.appreadstories.story.list.filter

import huce.fit.appreadstories.story.list.BaseStoryListPresenter

interface StoryFilterPresenter : BaseStoryListPresenter {
    fun setSpecies(species: String)
    fun setStatus(status: String)
    fun getData()
}